### PR TITLE
add serialization & deserialization

### PR DESCRIPTION
**Pull request by sofiaso**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/Tools/GetData/Urls.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿namespace TestFramework.Tools.deserialization
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Tools/GetData/Urls.cs:1]: Is this a correct namespace?

File: TAF_EasyRest/TestFramework/Tools/GetData/RoleCredentials.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿namespace TestFramework.Tools.deserialization
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Tools/GetData/RoleCredentials.cs:1]: Is this a correct namespace?

File: TAF_EasyRest/TestFramework/Tools/GetData/GetRoleCredentials.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿//using Newtonsoft.Json;
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Tools/GetData/GetRoleCredentials.cs:1]: Please remove this line.

**Review by LoveMyDaughter — APPROVED** [review]
Check namespaces.

**Review by dmatsev — APPROVED** [review]

**Pull request merged**